drafts/pair-programming-mistakes.md

```diff
@@ -0,0 +1,66 @@
+
+
+## Pair programming mistakes I made, so you don't have to
+I recently had the feeling that I'm missing out a lot of potential when pair programming. Identifying the exact issue was not immediate, but there was a clear sense that things could be better. When I dived into the topic, I realized that it's following the [Bushnell's Law](https://en.wikipedia.org/wiki/Bushnell%27s%5FLaw). While it's easy to understand the main concepts like driver - navigator, pair programming is a topic hard to master. If you're interested in the theory like different styles of pair programming, I can highly recommend the article ["on pair programming" from Martin Fowler's blog](https://martinfowler.com/articles/on-pair-programming.html), it provides an excellent entry to the topic. 
+
+As a prelude, I see pair programming as people's business and it is therefore highly subjective. The following examples are based on my own experiences and how I feel about them. 
+
+
+## Overwhelmed by the Driver's Pace
+In the early days of my programmer career, right after university, I found myself working with another newcomer to the team. We were both strangers to each other and the project, which added an extra layer of complexity to our collaboration.
+
+Our roles were clearly defined - he was the driver, I was the navigator. As we ventured into the new project and environment, I found his pace challenging to match. Since he was already a senior, he was swift in his actions, and I often found myself trailing behind, struggling to keep up.
+
+My unfamiliarity with him and the fear of appearing less competent held me back from asking him to slow down. This was further complicated by his eagerness to showcase his skills.
+
+![Overwhelmed by my pairing partner.](https://cdn.hashnode.com/res/hashnode/image/upload/v1717162511151/JGYtq53Fe.png?auto=format)
+
+Looking back, the solution to this problem is simpler than it seems. It’s crucial to remember that it’s okay not to know everything and that it’s encouraged to ask questions. It’s also about acknowledging the presence of [imposter syndrome](https://en.wikipedia.org/wiki/Impostor%5Fsyndrome) and fostering open communication. After all, pair programming is as much about learning as it is about coding. Effective communication forms the backbone of successful pair programming, making it a rewarding experience for both parties involved.
+
+
+## Guiding a Junior
+I recently had the situation where I pair programmed with a junior developer who had just joined our team. As a more experienced developer, I took on the role of the navigator while the junior, understandably nervous due to his newness to the team, was the driver. During our pairing session, I provided many tips and tricks to the junior, particularly on how to navigate within the IDE. However, I noticed that our progress slowed down significantly during our working session.
```

Comment:
I would move the "understandably nervous due to his newness to the team" subsentence out of this sentence. Make it clear who is the navigator and who is the driver, then you could quickly write about that he was nervous because he is new